### ISSUE TITLE
multiple instances of ffmpeg

### ISSUE THREAD

**Issue by lolitsme1** [fluent-ffmpeg/node-fluent-ffmpeg#884]
Does this module support multiple instances of ffmpeg?
Im trying to create a local streaming service with express. Im managing pipes in my backend, but whenever i try to create a second stream, the first one dies. Im keeping my fluent-ffmpeg instances in a object.
I can post my source code if you need it.

**Comment by davidjytang** [comment]
Yes multiple instances work. I was able to have hundreds of ffmpeg instance running at the same time.

**Comment by njoyard** [comment]
Indeed, there's no reason it would not work. There's most likely something in your code that makes your instances interact and fail.

**Comment by evilArsh** [comment]
> Yes multiple instances work. I was able to have hundreds of ffmpeg instance running at the same time.

why does the ffmpeg event 'start'  only be triggered once in multiple  instance?

when i initialize a second ffmpeg instance,the 'start' event would not be triggered

**Comment by afreidz** [comment]
> why does the ffmpeg event 'start' only be triggered once in multiple instance?
> 
> when i initialize a second ffmpeg instance,the 'start' event would not be triggered

yes!! this has been frustrating me for days! none of the events fire for any subsequent instance for me.

**Comment by felix-voicehub** [comment]
Same problem here! Nothing in for example `.on('progress')` and the other events fires after the first call.

@njoyard Any suggestions how this should be working properly?
